CHANGELOG.rst

```diff
@@ -55,7 +55,7 @@ Removals
 
 New Features
 ------------
-
+- Added a link to the VSO Health Report in the Fido Results when using the VSOClient. (`#7853 <https://github.com/sunpy/sunpy/issues/7853>`__)
```

Comment:
This is unfortunately not how changelog entries are created, please revert this file and follow https://github.com/sunpy/sunpy/blob/main/changelog/README.rst